### ISSUE TITLE
 Parts of equations being displayed as blocks.

### ISSUE THREAD

**Issue by janpfeifer** [arXiv/html_feedback#2316]
### Description

Equations (2), (3) and (4) display as blocks

### (Optional:) Please add any files, screenshots, or other information here.

![image](https://github.com/user-attachments/assets/f16a9317-4167-41c1-b0ce-fb5f543be5aa)

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

1348d4c7-4de3-40ff-a420-0973ed39b444

### Paper URL

https://arxiv.org/html/2305.17205v2

### Browser

Chrome/129.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
This is a good example of a non-standard color that needs to be remapped in dark mode. Thank you for the report.

**Comment by janpfeifer** [comment]
Indeed. I noticed it after I opened the PDF. It's no simple task converting colors to dark mode -- I've noticed Adobe Acrobat Reader also is not perfect in dark mode in the past (some years ago, not sure how it is today).

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved with today's update to our latexml toolchain.

We have regenerated the article and you should be able to see an improvement.
